src/frontend/komponenter/Fagsak/Vilkårsvurdering/Vilkårsvurdering.tsx

```diff
@@ -43,6 +57,12 @@ const Vilkårsvurdering: React.FunctionComponent<IProps> = ({ fagsak, åpenBehan
         vilkårsvurdering,
     } = useVilkårsvurdering();
     const { erLesevisning } = useBehandling();
+    const { oppdaterRegisteropplysninger } = useFagsakRessurser();
+    const { toggles } = useApp();
+
+    const skalViseRegisteropplysninger = toggles[ToggleNavn.skjønnsvurdering];
+    const registeropplysningerHentetTidpsunkt =
```

Comment:
Burde vi modelert dette litt annerledes i fremtiden. For eksempel kunne vi bygd opp en entitet `personGrunnlag` med `hentetTidspunkt` og `personer`.

Reply:
https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=Tea-5010